pkg/controllers/dynakube/otelc/statefulset/env.go

```diff
@@ -107,3 +120,33 @@ func getEnvs(dk *dynakube.DynaKube) []corev1.EnvVar {
 
 	return envs
 }
+
+func getProxyEnvValue(envVar string, src *value.Source) corev1.EnvVar {
+	if src.ValueFrom != "" {
+		return corev1.EnvVar{
+			Name: envVar,
+			ValueFrom: &corev1.EnvVarSource{
+				SecretKeyRef: &corev1.SecretKeySelector{
+					LocalObjectReference: corev1.LocalObjectReference{Name: src.ValueFrom},
+					Key:                  dynakube.ProxyKey,
+				},
+			},
+		}
+	}
+
+	return corev1.EnvVar{Name: envVar, Value: src.Value}
+}
+
+func getNoProxyValue(dk *dynakube.DynaKube) string {
```

Comment:
same here, it would be much easier if we name it accordingly.

Reply:
Renamed [7bc2317](https://github.com/Dynatrace/dynatrace-operator/pull/4630/commits/7bc23172d110807e4fb55fd5a450a88de1bae893)